app/services/users.js

```diff
@@ -1,5 +1,19 @@
 const { User } = require('../models');
+const logger = require('../logger');
+const errors = require('../errors');
 
-exports.emailExists = email => User.findOne({ where: { email } });
+exports.emailExists = email => {
```

Comment:
If you are returning the query response without mapping the result, you can write this function as:
```.js
exports.emailExists = email =>
  User.findOne({ where: { email } }).catch(error => {
    logger.error('Error while trying to get user by email', error.message);
    throw errors.databaseError(error.message);
  });
```